pkg/util/crypt/crypt_dev.go

```diff
@@ -299,6 +300,19 @@ func (crypt *Device) Open(key []byte, path string) (string, error) {
 
 			return "", fmt.Errorf("cryptsetup open failed: %s: %v", string(out), err)
 		}
+
+		for attempt := 0; true; attempt++ {
+			_, err := os.Stat("/dev/mapper/" + nextCrypt)
+			if err == nil {
+				break
+			}
+			delay := 100 * (1 << attempt) * time.Millisecond
+			if delay > 12800*time.Millisecond {
```

Comment:
If you can make the conditionals more explicit that the max wait here is 25.5s then that would be useful for future readers skimming over the code. Some concise comments about the exponential backoff progression / max would work also.

Reply:
Good idea. I've add another variable and comment to make the total delay so far calculation clear.